kms/encrypt_symmetric.go

```diff
@@ -53,8 +59,24 @@ func encryptSymmetric(w io.Writer, name string, message string) error {
 		return fmt.Errorf("failed to encrypt: %v", err)
 	}
 
+	// Optional, but recommended: perform integrity verification on result.
+	// For more details on ensuring E2E in-transit integrity to and from Cloud KMS visit:
+	// https://cloud.google.com/kms/docs/data-integrity-guidelines
+	if result.VerifiedPlaintextCrc32C == false {
+		return fmt.Errorf("Encrypt: request corrupted in-transit")
+	}
+	if crc32c(result.Ciphertext) != result.CiphertextCrc32C {
+		return fmt.Errorf("Encrypt: response corrupted in-transit")
+	}
+	// End integrity verification
+
 	fmt.Fprintf(w, "Encrypted ciphertext: %s", result.Ciphertext)
 	return nil
 }
 
+func crc32c(data []byte) uint32 {
+	t := crc32.MakeTable(crc32.Castagnoli)
+	return crc32.Checksum(data, t)
+}
```

Comment:
This is leading to a compiler error because this function is declared here and in decrypt_symmetric.go. So, perhaps we should inline it. I think you can make a single `Table` then reuse it when calling `Checksum`?

Reply:
np. I made it into a nested function that lives only within the scope of the sample function. 
I prefer to leave the function intact, since I expect the user will have such a function in a shared util library, and so they can just copy past the function as is without needing the think about how to create one themselves.